app/models/manageiq/providers/redhat/infra_manager/event_parser.rb

```diff
@@ -34,9 +34,7 @@ def self.event_to_hash(event, ems_id = nil)
     user_href = ems_ref_from_object_in_event(event[:user])
     username  = nil
     if ems && user_href
-      ems.with_provider_connection do |rhevm|
-        username = Ovirt::User.find_by_href(rhevm, user_href).try(:[], :user_name)
-      end
+      username = ems.ovirt_services.username_by_href(user_href)
```

Comment:
re-arranging the data in the event and preparing it for MiqEvent to store.

but that might get easier once the graph refresh is used - Ladas has some nice examples where he uses event data